### PR TITLE
Paper Component done

### PR DESCRIPTION
**Pull request by pranitchandel**

### CODE REVIEW

File: src/shared/atom/Paper/index.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+interface PaperGroupPropType {
+  paper: any;
+}
+
+const PaperComponent = (props: PaperGroupPropType) => {
+  const { paper } = props;
+  return (
+    <Box
+      sx={{
```
Comment (AbhijeetKumarGupta) [inline src/shared/atom/Paper/index.tsx:18]: Please make it dynamic, and add custom style prop

File: src/shared/atom/Paper/index.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        display: "flex",
+        "& > :not(style)": {
+          m: 1,
+          width: 128,
+          height: 128,
+        },
+      }}
+    >
+      {paper.map((papr: any) => (
+        <Paper {...papr}></Paper>
```
Comment (AbhijeetKumarGupta) [inline src/shared/atom/Paper/index.tsx:28]: please add key prop

**Review by AbhijeetKumarGupta — CHANGES_REQUESTED** [review]

File: src/shared/atom/Paper/index.tsx (hunk 2 above)
Comment (pranitchandel) [reply src/shared/atom/Paper/index.tsx]: done

File: src/shared/atom/Paper/index.tsx (hunk 1 above)
Comment (pranitchandel) [reply src/shared/atom/Paper/index.tsx]: done

**Pull request merged**